include/s3select.h

```diff
@@ -2420,6 +2492,13 @@ class base_s3object
         for (auto& a : *m_s3_select->get_aliases()->get())
         {
           a.second->invalidate_cache_result();
+          m_sa->set_column_name(a.first);
```

Comment:
why use the `scratch_area` for saving the column-names ?

Reply:
Can't use a base_s3object member here as set_column_name is used in s3select_oper.h

Reply:
Removed the dependencies on variable class in oper.h. Scratch area can be avoided now